src/common/OneDSLoggerTelemetry/oneDSLogger.ts

```diff
@@ -270,17 +271,17 @@ export class OneDSLogger implements ITelemetryLogger{
 
                     envelope.data.clientSessionId = vscode.env.sessionId;
                     envelope.data.vscodeSurface = getExtensionType();
-                    envelope.data.vscodeMachineId = vscode.env.machineId;
                     envelope.data.vscodeExtensionName = EXTENSION_ID;
                     envelope.data.vscodeExtensionVersion = getExtensionVersion();
                     envelope.data.vscodeVersion = vscode.version;
                     envelope.data.domain = vscode.env.appHost;
+                    envelope.data.measurements = envelope.data.measurement;
+                    envelope.data.timestamp = envelope.time;
```

Comment:
Lets remove the time. Not needed inside data

Reply:
@BidishaMS removed timestamp mapping